src/components/Contact/Contact.css

```diff
@@ -1,2 +1,44 @@
 .contact {
+  display: flex;
+  flex-direction: column;
+  top: 20%;
+  height: 80%;
+  width: 60%;
+  margin: auto;
+  position: relative;
+  align-items: center;
+  align-content: center;
+}
+
+.contact .contact-info {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.contact h2 {
+  font-weight: 900;
+}
+
+.google-map {
+  width: 60%;
+  height: 100%;
+  margin: 0;
+  border: 1px solid #aaa;
+  border-radius: 5px;
+}
+
+.contact .ul {
```

Comment:
Is this a typo? `.ul`

Reply:
oh yeah oops shit looks wack maybe forgot to save and pushed